cli/src/funTest/assets/git-repo-expected-repository.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Please describe briefly in the commit message in what way the test is simplified. I.e. only serialize `Repository` instead of an `OrtResult`, which avoids the need for patching (in that test case).